components/camel-jsonpath/src/main/java/org/apache/camel/jsonpath/JsonPathEngine.java

```diff
@@ -38,7 +41,8 @@
     private final Configuration configuration;
 
     public JsonPathEngine(String expression) {
-        this.configuration = Configuration.defaultConfiguration();
+        Defaults defaults = DefaultsImpl.INSTANCE;
```

Comment:
This comment is 2 years late, but I don't think there's any need to use json-path's internal API?

`Configuration.Builder.build()` will do the same thing internally anyway if no custom `jsonProvider` provided.

Reply:
@yihtserns you are welcome to provide a PR to fix this

Reply:
Thanks I just pushed a fix for this: https://github.com/apache/camel/commit/9e58df3c602f2502333ca0760e1a0baee1c5a909

Reply:
I wanted to create a PR, but I'm not very sure of the process.

I read that I have to:
1. Create a JIRA ticket
2. Commit and reference that ticket ID in the message
3. Create a PR

Are the steps correct?  And if it is, will that be too much for some refactoring to remove some duplicated lines?

Reply:
If its just some cleanup and minor code changes then you do NOT need to create a JIRA ticket. We only use JIRA for normal/bigger things. So in your case, just do the PR

Reply:
👌